lib/services/form_validators.dart

```diff
@@ -1,34 +1,51 @@
 class FormValidators {
   static String emailRegex = r"^[a-zA-Z0-9.]+@[a-zA-Z0-9]+\.[a-zA-Z]+";
+  static String loginRegex = r"^[a-z0-9_.-]*$";
   static var _validatorConstants = {
-    'empty_email'    : 'Enter email',
-    'invalid_email'  : 'Invalid email',
-    'emply_password' : 'Enter password',
+    'empty_email': 'Enter email',
+    'empty_login': 'Enter email or username',
+    'emply_password': 'Enter password',
+    'invalid_email': 'Invalid email',
+    'invalid_username': 'Invalid username',
```

Comment:
Add to all constants 'Please enter a valid ...'

Reply:
Done